GENERATORS/SIMULATIONS/SECOND_ORDER_SYSTEM/SECOND_ORDER_SYSTEM.py

```diff
@@ -1,13 +1,13 @@
 import numpy as np
-from flojoy import flojoy, OrderedPair, DefaultParams, SmallMemory
+from flojoy import flojoy, OrderedPair, DefaultParams, SmallMemory, Vector
 
 
 memory_key = "SECOND_ORDER_SYSTEM"
 
 
 @flojoy(inject_node_metadata=True)
 def SECOND_ORDER_SYSTEM(
-    default: OrderedPair,
+    default: Vector,
```

Comment:
Does this make sense to be used with only a vector? I'm not sure.. Since this is a second order differential equation it needs `x` values as well right?

Reply:
what was passed to x was `default.y` so I assumed it would be default.v. What are your thoughts? should I change it back to x?
<img width="557" alt="Screenshot 2023-07-14 at 7 58 18 PM" src="https://github.com/flojoy-io/nodes/assets/83793112/c490527e-645d-44ed-9d1b-71af6ba0d1eb">

Reply:
perhaps you should ask the person who wrote this node originally (I assume it was dallas or tristan)